Presentation/MasterDetailSelection.swift

```diff
@@ -116,7 +121,7 @@ public final class MasterDetailSelection<Elements: BidirectionalCollection>: Sig
             guard let `self` = self else {
                 return NilDisposer()
             }
-            onSet(self.isCollapsed.value ? nil : self.current)
+            onSet((self.isCollapsed.value == nil || self.isCollapsed.value == true) ? nil: self.current)
```

Comment:
```suggestion
            onSet(self.isCollapsed.value != false ? nil : self.current)
```

Reply:
So I was reading this condition when I wrote it this way and I thought that it's not clear enough so then I added a comment which basically said "if it's not determined or it's collapsed" but that's redundant if I write it this way. Is it ok if I leave it this way for clarity?

Reply:
Sure, keep it!